opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/HeatMapRestService.java

```diff
@@ -51,7 +39,17 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
-import com.google.common.collect.Lists;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
```

Comment:
Please configure your IDE to match with the ecilpse import pattern to avoid unnecessary changes.